codefori/vscode-ibmi
# Add `Download Logs` option to Help and Support view

### Changes

To help with gathering of logs, I added a new `Download Logs` option to the `Help and Support` view when connected to an IBM i. This will prompt users to select from different logs to download and the location where they would like to download it to. The files will be put into a zip with the following format: `CodeForIBMi_YYYYMMDD_HHMMSS.zip`. Note that if a log file is not found, the option will not be shown in the quick pick.

This should hopefully make it easier for users to share their logs in GitHub Issues instead of manually having to retrieve and copy them. Should also prevent the back and forth of explaining where to grab these logs :smiley:

Several new strings were added as well which will need to be translated.

![image](https://github.com/user-attachments/assets/f2b05239-1c67-4ac4-9d50-dafea586597b)

### How to test this PR

Examples:

1. Connect to an IBM i.
2. Click the `Download Logs` option.
3. Select the logs you want to download and then where you would like to download it to.
4. Verify the logs are downloaded.
5. Disconnect and verify the option is no longer visible.

### Checklist
